tools/ci_build/github/azure-pipelines/templates/win-ci-2019.yml

```diff
@@ -151,15 +151,16 @@ jobs:
           arguments: '--configuration $(BuildConfig) -p:Platform="Any CPU" -p:OnnxRuntimeBuildDirectory="$(Build.BinariesDirectory)" -p:OrtPackageId=${{ parameters.OrtPackageId }}'
           workingDirectory: '$(Build.SourcesDirectory)\csharp'
 
-    - ${{ if and(eq(parameters.BuildCSharp, true), in(parameters['sln_platform'], 'Win32', 'x64')) }}:
-      - task: DotNetCoreCLI@2
-        displayName: 'Test C#'
-        inputs:
-          command: test
-          projects: '$(Build.SourcesDirectory)\csharp\test\Microsoft.ML.OnnxRuntime.Tests\Microsoft.ML.OnnxRuntime.Tests.csproj'
-          configuration: '$(BuildConfig)'
-          arguments: '--configuration $(BuildConfig) -p:Platform="Any CPU" -p:OnnxRuntimeBuildDirectory="$(Build.BinariesDirectory)" -p:OrtPackageId=${{ parameters.OrtPackageId }}'
-          workingDirectory: '$(Build.SourcesDirectory)\csharp'
+    - ${{ if in(parameters['sln_platform'], 'Win32', 'x64') }}:      
+      - ${{ if eq(parameters.BuildCSharp, true) }}:
```

Comment:
why is this different from the original condition?

Reply:
it's different to Line 165